src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,33 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import classNames from 'classnames';
+
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    title,
+    completed,
+    user,
+  } = todo;
+
+  return (
+    <article
+      className={
+        classNames(
+          'TodoInfo',
+          { 'TodoInfo--completed': completed },
+        )
+      }
```

Comment:
a bit of an overkill
```suggestion
      className={classNames(
        'TodoInfo',
        { 'TodoInfo--completed': completed },
      )}
```